### PR TITLE
Issue #6 add ci for fedora iso

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

**Comment by praveenkumar** [comment]
retest this please.

**Comment by praveenkumar** [comment]
retest this please.

**Comment by praveenkumar** [comment]
retest this please.

**Comment by praveenkumar** [comment]
retest this please.

File: Vagrantfile [hunk 1]
```diff
@@ -23,11 +23,14 @@ Vagrant.configure(2) do |config|
     target_path = ENV['USERPROFILE'].gsub(/\\/,'/').gsub(/[[:alpha:]]{1}:/){|s|'/' + s.downcase.sub(':', '')}
     config.vm.synced_folder ENV['USERPROFILE'], target_path, type: 'sshfs', sshfs_opts_append: '-o umask=000 -o uid=1000 -o gid=1000'
   else
-    config.vm.synced_folder ENV['HOME'], ENV['HOME'], type: 'sshfs', sshfs_opts_append: '-o umask=000 -o uid=1000 -o gid=1000'
+    config.vm.synced_folder "/opt/fedora", "/opt", type: 'sshfs', sshfs_opts_append: '-o umask=000 -o uid=1000 -o gid=1000'
```
Comment (gbraad) [inline Vagrantfile]: why `opt`?

**Pull request merged**